schema/objects/transactions/issuance/EquityCompensationIssuance.schema.json

```diff
@@ -58,9 +58,18 @@
       "$comment": "REQUIRED in v2"
     },
     "vesting_terms_id": {
-      "description": "Identifier of the VestingTerms to which this security is subject.  If not present, security is fully vested on issuance.",
+      "description": "Identifier of the VestingTerms to which this security is subject. If neither `vesting_terms_id` or `vestings` are present then the security is fully vested on issuance.",
```

Comment:
Now that we have the validator project, when we write natural language rules like this, let's also open a matching issue in the validator repo and cross-link the issues. That way we can finally start tracking these sorts of things. Even if it takes some time to get the rule into the XState code, we'll be able to keep track.

Reply:
Good idea.
opened https://github.com/Open-Cap-Table-Coalition/OCF-XState-Validation/issues/76